### PR TITLE
feat: Claim name from deploy worlds

### LINKED ISSUE #2584: Claim name from scene publish
## 🎉 Description

Add posiblity to claim a name from the publish to world flow
* [ ] 🐛 This is a bug report.
* [x] 📈 This is a feature request.

## 📝 Details
* [x] There should be an option in the names dropdown to claim a new name
* [x] When clicking claim name it should redirect to claim name section.
* [ ] When finishing claiming name or canceling claim flow the user should go back to publish scene page with publish to world modal open

### PR DESCRIPTION
**Pull request by cyaiox**
This PR adds a claim name page redirection from the deploy to worlds modal

Closes #2584 

Depends on: #2626

### CODE REVIEW

File: src/components/Modals/DeployModal/DeployToWorld/DeployToWorld.tsx [hunk 1]
```diff
@@ -270,7 +272,7 @@ export default function DeployToWorld({
           <span>{t('deployment_modal.deploy_world.description')}</span>
         </div>
         <div className={styles.modalForm}>
-          {thumbnailUrl ? renderThumbnail() : null}
+          {project?.thumbnail ? renderThumbnail() : null}
```
Comment (cyaiox) [inline src/components/Modals/DeployModal/DeployToWorld/DeployToWorld.tsx:275]: not related to this change but remove an unnecessary variable

File: src/components/ClaimENSPage/ClaimENSPage.container.ts [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
 import { getData as getWallet, getMana } from 'decentraland-dapps/dist/modules/wallet/selectors'
 import { openModal } from 'modules/modal/actions'
 import { RootState } from 'modules/common/types'
+import { hasHistory } from 'modules/location/selectors'
+import { DeployToWorldLocationStateProps, FromParam } from 'modules/location/types'
 import { MapDispatchProps, MapDispatch, MapStateProps } from './ClaimENSPage.types'
 import ClaimENSPage from './ClaimENSPage'
 
 const mapState = (state: RootState): MapStateProps => {
+  const isFromDeployToWorld = (getLocation(state).state as DeployToWorldLocationStateProps)?.fromParam === FromParam.DEPLOY_TO_WORLD
```
Comment (meelrossi) [inline src/components/ClaimENSPage/ClaimENSPage.container.ts:12]: do we want to maintain this redirection if the user refreshes the page? maybe add it as query param?

File: src/modules/location/sagas.ts [hunk 3]
```diff
@@ -37,8 +37,22 @@ function* handleSetENSContentSuccess(action: SetENSContentSuccessAction) {
   }
 }
 
-function* goToActivity() {
-  yield put(replace(locations.activity()))
+function* goToActivity(action: ClaimNameSuccessAction | null = null) {
```
Comment (meelrossi) [inline src/modules/location/sagas.ts:40]: maybe rename this? so its not called goToActivity?

Comment (cyaiox) [reply src/modules/location/sagas.ts:40]: makes sense, I'll rename it

**Pull request merged**